src/app/components/Transcript/index.tsx

```diff
@@ -0,0 +1,77 @@
+/** @jsx jsx */
+/* eslint-disable jsx-a11y/aria-role */
+import { jsx } from '@emotion/react';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import { useContext } from 'react';
+import styles from './index.styles';
+import Text from '../Text';
+import TranscriptTimestamp from './TranscriptTimestamp';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+import { RightArrow as ArrowSvg } from '../icons';
+import { TranscriptBlock, TranscriptItem } from './types';
+
+const DEAFULT_TRANSLATIONS = {
+  readTranscript: 'Read transcript',
+  disclaimer:
+    ' This transcript has been reviewed by a journalist, it was generated with AI (artificial intelligence).',
```

Comment:
```suggestion
    'This transcript has been reviewed by a journalist, it was generated with AI (Artificial Intelligence).',
```

Reply:
Or should we put the AI in brackets?
```suggestion
    'This transcript has been reviewed by a journalist, it was generated with Artificial Intelligence (AI).',
```